drivers/dma/dmamux_stm32.c

```diff
@@ -230,15 +232,23 @@ static const struct dma_driver_api dma_funcs = {
 #define DMAMUX_CHANNELS_INIT_1(count) \
 	UTIL_LISTIFY(count, INIT_DMAMUX_1_CHANNEL)
 
+
+#if defined(CONFIG_SOC_SERIES_STM32G0X)
```

Comment:
In that case, something like follows could be used:
```
COND_CODE_1(DT_INST_NODE_HAS_PROP(index, clocks), ....
```

Reply:
done